src/capture_RPiHQ.cpp

```diff
@@ -395,19 +427,30 @@ int RPiHQcapture(int asiAutoFocus, int asiAutoExposure, int asiExposure, int asi
 		if (myModeMeanSetting.mode_mean) {
 			ss.str("");
 			ss << myRaspistillSetting.analoggain;
-			command += " --analoggain " + ss.str();
+			if (libcamera)
+				command += " --gain " + ss.str();
+			else
+				command += " --analoggain " + ss.str();
 
+//xxxx libcamera just has "gain".  If it's higher than what the camera supports,
```

Comment:
I think we can remove digital gain. My tests gave only bad results

Reply:
How about if we do that in a new PR?  It'll impact a few files and I'd like to get this PR out first.

Reply:
Yes, this was only a comment.

Yesterday i did some tests with libcamera. The following came out

1. gain values ​​from 1-89.0435 possible
2. ag and dg are chosen automatically
3. Pictures were grainier than with raspistill (test setup - no starry sky possible)
4. looks like ag has a limit of 22.2609
```

libcamera-still -o long_exposure.jpg --shutter 1000 --gain 10 --awbgains 2,2 --immediate -v

gain: / libcamera debug output
0.5:	Ag 1 Dg 1 Total 1
1:	Ag 1 Dg 1 Total 1
2: 	Ag 2 Dg 1 Total 2
4: 	Ag 4 Dg 1 Total 4
5:	Ag 4.99512 Dg 1.00098 Total 5
6:	Ag 5.9883 Dg 1.00195 Total 6
7:	Ag 6.96599 Dg 1.00488 Total 7
8:	Ag 8 Dg 1 Total 8
10: 	Ag 9.94175 Dg 1.00586 Total 10
16:	Ag 16 Dg 1 Total 16
20: 	Ag 19.6923 Dg 1.01562 Total 20
32:	Ag 22.2609 Dg 1.4375 Total 32
50: 	Ag 22.2609 Dg 2.24609 Total 50
100: 	Ag 22.2609 Dg 4 Total 89.0435
```